src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
 // write your code here
+
+const x = document.querySelector('thead');
+
+for (const ch of x.children) {
+  ch.append(ch.children[1].cloneNode(true));
+  ch.insertBefore(ch.children[5], ch.children[3]);
+}
+
+const y = document.querySelector('tbody');
+
+for (const ch of y.children) {
+  ch.append(ch.children[1].cloneNode(true));
+  ch.insertBefore(ch.children[5], ch.children[3]);
+}
+
+const z = document.querySelector('tfoot');
+
+for (const ch of z.children) {
+  ch.append(ch.children[1].cloneNode(true));
+  ch.insertBefore(ch.children[5], ch.children[3]);
+}
```

Comment:
You need only one loop to complete this task, just get all tr elements on the page and loop through them